packages/system/src/color.ts

```diff
@@ -55,23 +54,7 @@ export const color = (props: ColorProps): ResponsiveStyle => {
     const cssValue = props[key as ColorKeys];
     if (cssValue != undefined) {
       const property = colorMappings[key as ColorKeys];
-      const userTheme = theme.getUserTheme();
-      let converter: (value: string | number) => string | number;
-      if (userTheme.colors) {
-        converter = (value) => {
-          if (value in (userTheme.colors ?? {})) {
-            return userTheme.colors?.[value] as string;
-          }
-          return value;
-        };
-      } else {
-        converter = (v) => v;
-      }
-      const responsiveStyles = applyResponsiveStyles(
-        property,
-        cssValue,
-        converter
-      );
```

Comment:
For the same reasons as the previous comment, I would like you to revert the changes to this code as well.